matrix-sdk-android/src/test/java/org/matrix/android/sdk/test/fakes/FakeRealmConfiguration.kt

```diff
@@ -33,9 +33,9 @@ internal class FakeRealmConfiguration {
     val instance = mockk<RealmConfiguration>()
 
     fun <T> givenAwaitTransaction(realm: Realm) {
-        val transaction = slot<suspend (Realm) -> T>()
-        coEvery { awaitTransaction(instance, capture(transaction)) } coAnswers {
-            secondArg<suspend (Realm) -> T>().invoke(realm)
+        val transaction = slot<(Realm) -> T>()
```

Comment:
I realized the capture of the second argument is in fact not necessary when mocking the answer, so we could simplify this by:
```
fun <T> givenAwaitTransaction(realm: Realm) {
    coEvery { awaitTransaction(instance, any<(Realm) -> T>()) } answers {
        secondArg<(Realm) -> T>().invoke(realm)
    }
}
```

Reply:
indeed